src/classes/CON_PrimaryAffls_TDTM.cls

```diff
@@ -51,151 +51,200 @@ public class CON_PrimaryAffls_TDTM extends TDTM_Runnable {
+        TDTM_ProcessControl.setRecursionFlag(TDTM_ProcessControl.registeredTrigger.CON_PrimaryAffls_TDTM_After_Update, false);
+    }
+
+    /*******************************************************************************************************
+    * @description Returns if Contact Deletion Prevention custom setting is not enabled or
+    * Passes a map of contacts By Id to SRVC_Contact_PreventDeletion and returns a List of Contacts that have
+    * been prevented for deletion
+    * @param newSObjectList list of sObject records that are being deleted
+    * @return void
+    ********************************************************************************************************/
+    @TestVisible
+    private void handleAfterInsert(List<SObject> newSObjectList) {
+
+        if (this.isInsertRecursion() == true) {
+            return;
         }
+
+        setInsertRecursion();
+
+        Map<Id, Contact> newContactById = new Map<Id, Contact>((List<Contact>)newSObjectList);
+
+        // Get Primary Affiliation Mapping Models from the Primary Affiliation Settings Mapper
+        List<MODL_PrimaryAfflMapper> primaryAffiliationModels = this.locateAffiliationSettingMapper().getModelsFromSettings();
+
+        // Get the Affiliation records to create from the Contact Primary Affiliation Service
+        List<Affiliation__c> affiliationsToInsert = this.locateContactAffiliationService().getAffiliationsToCreate(newContactById.values(), primaryAffiliationModels);
+
+        if (affiliationsToInsert.size() == 0) {
+            this.unsetInsertRecursion();
+            return;
+        }
+
+        this.setAffiliationUpdateRecursion();
+
+        DmlWrapper dmlWrapper = new DmlWrapper();
+        dmlWrapper.objectsToInsert.addAll((List<SObject>)affiliationsToInsert);
+        TDTM_TriggerHandler.processDML(dmlWrapper, true);
+        this.resetAffiliationUpdateRecursion();
+        this.unsetInsertRecursion();
+
     }
 
-    private List<Affiliation__c> makeOtherAfflsNotPrimary(Map<ID, List<ID>> changedConToAccsMap) {
-        //Find all the affiliations to make non-primary.
-        List<Affiliation__c> afflsToMakeNonPrimary = new List<Affiliation__c>();
+    /*******************************************************************************************************
+    * @description Returns if Contact Deletion Prevention custom setting is not enabled or
+    * Passes a map of contacts By Id to SRVC_Contact_PreventDeletion and returns a List of Contacts that have
+    * been prevented for deletion
+    * @param oldSObjectList list of sObject records that are being deleted
+    * @return List<Contact>
+    ********************************************************************************************************/
+    @TestVisible
+    private void handleAfterUpdate(List<SObject> oldSObjectList, List<SObject> newSObjectList) {
+
+        if (this.isUpdateRecursion() == true) {
+            return;
+        }
         
```

Comment:
seems to have unnecessary double spacing here!

Reply:
hmm looks good to me, not sure if I am missing anything.

Reply:
updated.